timber/build.gradle

```diff
@@ -31,11 +31,13 @@ configurations {
 }
 
 dependencies {
+  compile 'org.jetbrains:annotations:13.0'
```

Comment:
`compile` pushes the dependency on all consumers of the library which we should not do. `provided` allows the dependency to be used without forcing downstream apps to bundle these annotations.